{
  "dataType": "CVE_RECORD",
  "containers": {
    "adp": [
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2024-31217",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "poc"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "partial"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-06-12T15:53:55.205861Z"
              }
            }
          }
        ],
        "affected": [
          {
            "cpes": [
              "cpe:2.3:a:strapi:strapi:*:*:*:*:*:*:*:*"
            ],
            "vendor": "strapi",
            "product": "strapi",
            "versions": [
              {
                "status": "affected",
                "version": "0",
                "lessThan": "4.22.0",
                "versionType": "custom"
              }
            ],
            "defaultStatus": "unknown"
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-06-12T15:55:44.339Z"
        }
      }
    ],
    "cna": {
      "title": "@strapi/plugin-upload has a Denial-of-Service via Improper Exception Handling",
      "source": {
        "advisory": "GHSA-pm9q-xj9p-96pm",
        "discovery": "UNKNOWN"
      },
      "metrics": [
        {
          "cvssV3_1": {
            "scope": "UNCHANGED",
            "version": "3.1",
            "baseScore": 5.3,
            "attackVector": "NETWORK",
            "baseSeverity": "MEDIUM",
            "vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:N/I:N/A:H",
            "integrityImpact": "NONE",
            "userInteraction": "NONE",
            "attackComplexity": "HIGH",
            "availabilityImpact": "HIGH",
            "privilegesRequired": "LOW",
            "confidentialityImpact": "NONE"
          }
        }
      ],
      "affected": [
        {
          "vendor": "strapi",
          "product": "strapi",
          "versions": [
            {
              "status": "affected",
              "version": "< 4.22.0"
            }
          ]
        }
      ],
      "references": [
        {
          "url": "https://github.com/strapi/strapi/security/advisories/GHSA-pm9q-xj9p-96pm",
          "name": "https://github.com/strapi/strapi/security/advisories/GHSA-pm9q-xj9p-96pm",
          "tags": [
            "x_refsource_CONFIRM"
          ]
        },
        {
          "url": "https://github.com/strapi/strapi/commit/a0da7e73e1496d835fe71a2febb14f70170135c7",
          "name": "https://github.com/strapi/strapi/commit/a0da7e73e1496d835fe71a2febb14f70170135c7",
          "tags": [
            "x_refsource_MISC"
          ]
        }
      ],
      "descriptions": [
        {
          "lang": "en",
          "value": "Strapi is an open-source content management system. Prior to version 4.22.0, a denial-of-service vulnerability is present in the media upload process causing the server to crash without restarting, affecting either development and production environments. Usually, errors in the application cause it to log the error and keep it running for other clients. This behavior, in contrast, stops the server execution, making it unavailable for any clients until it's manually restarted. Any user with access to the file upload functionality is able to exploit this vulnerability, affecting applications running in both development mode and production mode as well. Users should upgrade @strapi/plugin-upload to version 4.22.0 to receive a patch.\n"
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "CWE",
              "cweId": "CWE-248",
              "description": "CWE-248: Uncaught Exception"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
        "shortName": "GitHub_M",
        "dateUpdated": "2024-06-12T14:50:37.999Z"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2024-31217",
    "state": "PUBLISHED",
    "dateUpdated": "2024-06-12T15:55:51.664Z",
    "dateReserved": "2024-03-29T14:16:31.901Z",
    "assignerOrgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
    "datePublished": "2024-06-12T14:50:37.999Z",
    "assignerShortName": "GitHub_M"
  },
  "dataVersion": "5.1"
}